locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/networking/SharedLocationSecurityGroupCustomizer.java

```diff
@@ -74,6 +74,9 @@
 
     private RangeSet<Integer> tcpPortRanges;
     private RangeSet<Integer> udpPortRanges;
+    private Boolean openIcmp;
```

Comment:
Looks like this fails on Azure - because Azure SGs don't support ICMP - probably should add a comment documenting what will happen in this case